tests/prepare/feature/crb/main.fmf

```diff
@@ -0,0 +1,4 @@
+summary: Common feature to enable/disable CRB
+description: Make sure enable/disable CRB works.
+tag+:
+  - provision-container
```

Comment:
```suggestion
  - provision-container
  - provision-only
```

We don't need to run the test as part of non-provision plans. Without `provision-only` it would be also executed under `/plans/features/steps/prepare`, not only under `/plans/provision/container/the-rest` as expected.